pkg/kudoctl/cmd/init/crds.go

```diff
@@ -31,6 +35,20 @@ func installCrds(client apiextensionsclient.Interface) error {
 	return nil
 }
 
+func validateCrd(client v1beta1.CustomResourceDefinitionsGetter, crd *apiextv1beta1.CustomResourceDefinition) error {
+	existingCrd, err := client.CustomResourceDefinitions().Get(crd.Name, v1.GetOptions{})
+	if err != nil {
+		if os.IsTimeout(err) {
```

Comment:
I see the reasoning for this in `pkg/kudoctl/util/kudo/kudo.go` but I don't agree with it: IMO, it's valuable information the retrieving a CRD failed due to a timeout. While the underlying issue causing the timeout is indeed not CRD specific, it is the retrieval of the CRD that is affected by the timeout. I agree, that we shouldn't wrap the timeout, but this can be achieved by not wrapping in the `fmt.Errorf` below.
Another liability this brings is that this special treatment of timeouts needs to be done when bubbling up these errors: It needs to be done here, again in `kudo.NewManager`.

Though, for now we can stay consistent with the patterns introduced in `kudo.go` and keep this as-is.

Reply:
Yeah, let's keep this as it is right now, there was a discussion on https://github.com/kudobuilder/kudo/pull/1097 that I missed to review, will discuss this outside of this PR.

Reply:
Yeah. I'm not happy with this either. I've added this after I merged the changes from master, but I don't like it. We might/should change this later on, for now it's that way, i guess :-/